feature/gnoi/packet_link_qualification/tests/packet_link_qualification_test/packet_link_qualification_test.go

```diff
@@ -191,6 +194,17 @@ func TestListDelete(t *testing.T) {
 	}
 }
 
+// configInterfaceMTU configures interface MTU.
```

Comment:
I'd drop this comment. godoc comments are unnecessary on non-exported functions, and this comment is redundant just with the name.

Reply:
Removed the comment. thanks.